circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   node:
-   version: 0.12.2
```

Comment:
@rodrigopavezi Yeah it did, we'll be adding it to repos on a case by case basis I guess